packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -149,8 +145,7 @@ export default defineComponent({
       spaces,
       graphClient,
       loadResourcesTask,
-      areResourcesLoading,
-      accessToken,
+            accessToken,
```

Comment:
Indentation looks wrong. Linter should (hopefully) complain about this as well 😁